app/scripts/directives/raml-editor-help-button.js

```diff
@@ -0,0 +1,42 @@
+(function () {
+  'use strict';
+
+  angular.module('ramlEditorApp')
+    .directive('ramlEditorHelpButton', function ramlEditorHelpButton(
+      $window,
+      $timeout,
+      $modal
+    ) {
+      return {
+        restrict: 'E',
+        templateUrl: 'views/menu/help-menu.tmpl.html',
+        link:     function(scope) {
+          scope.contextHelpMenuOpen = false;
+
+          scope.openHelpContextMenu = function () {
+            $timeout(function () {
+              $window.addEventListener('click', function self () {
+                scope.$apply(function () {
+                  scope.contextHelpMenuOpen = false;
+                });
+
+                $window.removeEventListener('click', self);
+              });
+            });
+
+            scope.contextHelpMenuOpen = true;
+          };
+
+          scope.openHelpModal = function openHelpModal() {
```

Comment:
This code exists in two places now, probably it would be good to actually create a small HelpService and change both these calls.